src/commands/slow-mode.ts

```diff
@@ -0,0 +1,599 @@
+import { SlowMode, SlowModeStage } from '@/models/slow-mode';
+import handleSlowMode from '@/slow-mode';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { sendEventLogMessage } from '@/util';
+import { ChannelType, EmbedBuilder, type ChatInputCommandInteraction } from 'discord.js';
+
+async function slowModeHandler(interaction: ChatInputCommandInteraction): Promise<void> {
+	switch (interaction.options.getSubcommandGroup()) {
+		case 'stage': {
+			switch (interaction.options.getSubcommand()) {
+				case 'set':
+					return setStageHandler(interaction);
+				case 'unset':
+					return unsetStageHandler(interaction);
+			}
```

Comment:
Nesting `switch` statements like this is a bit much imo. There's only 2 options per case, this can easily be done as an `if else`